requirements.txt

```diff
@@ -2,3 +2,4 @@ setuptools
 psutil
 click
 distro
+requests
```

Comment:
Let's add `requests` to `snap/snapcraft.yaml` as otherwise package won't be build-able using `snapcraft`.

I.e:

```
parts:
  auto-cpufreq:
    plugin: python
    python-packages:
       - setuptools
       - wheel
       - requests
```

Should fix it.

Reply:
okay, i didnt altered the file, since I had no idea how it worked, let me make changes to it

Reply:
Basically, if all changes are setup correctly if you ran `snapcraft` you should end up with `auto-cpufreq_1.9.8_amd64.snap` in auto-cpufreq dir.